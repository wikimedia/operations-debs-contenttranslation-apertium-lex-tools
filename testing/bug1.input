^garrantzi<n>/importancia<n><f><ND>/trascendencia<n><f><ND>$

