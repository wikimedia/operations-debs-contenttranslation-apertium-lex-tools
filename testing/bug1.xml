<rules>
<rule comment="193"><match lemma="garrantzi" tags="*"><select lemma="importancia"/></match></rule>
<rule comment="7"><match lemma="garratz" tags="*"><select lemma="Ã¡cido"/></match></rule>
</rules>
