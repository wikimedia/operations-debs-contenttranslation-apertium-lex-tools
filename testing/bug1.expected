^garrantzi<n>/importancia<n><f><ND>$

